test/controllers/actions/AuthenticatedIdentifierActionSpec.scala

```diff
@@ -33,7 +33,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class AuthenticatedIdentifierActionSpec extends ControllerBaseSpec with MockAuthConnector {
```

Comment:
Just questioning why we kept this name if we changed the main controller to AllRolesAuthenticatedAction ?

Reply:
Missed them, will do now